### ISSUE TITLE
Ubuntu 14.04 make error

### ISSUE THREAD

**Issue by vire** [facebook/watchman#78]
running `make` leads into the following error

```
root.c: In function ‘stat_path’:
root.c:979:1: error: label ‘out’ defined but not used [-Werror=unused-label]
```

with configuration

```
        CPPFLAGS =  -D_REENTRANT -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE
        CFLAGS = -g -O2 -Wall -Wextra -Wdeclaration-after-statement -g -gdwarf-2 -fno-omit-frame-pointer
        LDFLAGS = 
        prefix: /usr/local
        version: 3.0.0
```

Linux 6430U 3.13.0-37-generic #64-Ubuntu SMP Mon Sep 22 21:28:38 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux

Any ideas what could be wrong?

**Comment by krzysiek1507** [comment]
Travis CI build failed too. Checkout to v3.0.0 and then run make.

**Comment by sid0** [comment]
Fixed with a183e075d0bc8da36436c0a046ba2b7af5412bd5. Thanks for reporting.
